week3/labs/lab3-1/public/js/a4.js

```diff
@@ -0,0 +1,35 @@
+var numbers = [1,2,3,4,5,6,7,8,9,10,11]
+
+for (i = 0; i < 11; i++){
+  console.log("i = " + i);
+}
+
+numbers.forEach(function(e){
+  console.log("number " + e);
+})
+
+for (i = 0; i < numbers.length; i++){
+  if (numbers[i] > 5)
+    console.log("larger than 5: " + numbers[i]);
+}
+
+numbers.forEach(function(e){
+  if (e > 5)
+    console.log("larger than 5: " + e);
+})
+
+console.log(numbers.pop());
+
+console.log(numbers.reverse());
+console.log(numbers.slice().reverse());
```

Comment:
for those statements, they may appear nice if you add a little English text to the front:

```
console.log("slice then reverse: ", numbers.slice().reverse())
```